summingbird-core/src/main/scala/com/twitter/summingbird/memory/Memory.scala

```diff
@@ -16,6 +16,9 @@
 
 package com.twitter.summingbird.memory
 
+import java.util
+
+import com.twitter.algebird.Monoid
```

Comment:
My bad, have been relying on intellij to eliminate these. intellij gave up at some point and I didn't relize. Removed now.